src/Microsoft.DotNet.Darc/src/DarcLib/Models/VirtualMonoRepo/SourceManifest.cs

```diff
@@ -0,0 +1,83 @@
+using Microsoft.DotNet.DarcLib.Helpers;
+using System;
+using System.Collections.Generic;
+using System.IO;
+using System.Linq;
+using System.Text.Json;
+
+namespace Microsoft.DotNet.Darc.Models.VirtualMonoRepo
```

Comment:
There is a new, better way to declare namespaces, Visual Studio should give you a refactoring for this line (`Ctrl+.`) called "Convert to file-scope namespaces" which looks like this:

```
namespace Microsoft.DotNet.Darc.Models.VirtualMonoRepo
Write;

class {
...
}
```